### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by tylerbutler**

### CODE REVIEW

File: docs/content/_index.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ title: "Fluid Framework"
 
   {{< cta
     linkClasses="play"
-    linkURL="/playground/"
+    linkURL="/docs/start/examples/"
     iconClasses="play"
-    linkText="Play with Fluid"
+    linkText="Examples"
```
Comment (tylerbutler) [inline docs/content/_index.md:29]: I'm not sure what else to put here now...

Comment (skylerjokiel) [reply docs/content/_index.md:29]: This seems reasonable for now.

**Review by skylerjokiel — APPROVED** [review]
There's a few more things that need updated.

On the examples page 

- `Click here to try out Brainstorm in the Playground.` needs to be removed
- Also the outdated article warning needs to go away

**Comment by tylerbutler** [comment]
The link to the playground was already removed. Also removed the outdated.

**Pull request merged**